connection-api/src/test/java/org/terracotta/exception/RuntimeEntityExceptionTest.java

```diff
@@ -57,11 +57,11 @@ public void testEntityInfoProvided() {
     String clz = "org.terracotta.SuperEntity";
     String name =  "test";
     RuntimeEntityExceptionImpl impl = new RuntimeEntityExceptionImpl(clz, name, description);
-    Assert.assertNotEquals(description, impl.getMessage());
-    Assert.assertEquals(clz, impl.getClassName());
-    Assert.assertEquals(name, impl.getEntityName());
-    Assert.assertTrue(impl.getMessage().contains(clz));
-    Assert.assertTrue(impl.getMessage().contains(name));
+    Assertions.assertNotEquals(description, impl.getMessage());
+    Assertions.assertEquals(clz, impl.getClassName());
+    Assertions.assertEquals(name, impl.getEntityName());
+    Assertions.assertTrue(impl.getMessage().contains(clz));
+    Assertions.assertTrue(impl.getMessage().contains(name));
```

Comment:
Static imports would look nicer.

Reply:
Fixed

Reply:
Dependency on multiple versions of Junit will exists temporarily, only till we migrate the Galvan tests as well.